cfgov/ask_cfpb/tests/models/test_pages.py

```diff
@@ -828,21 +828,19 @@ def test_category_page_context_es(self):
 
     @mock.patch('ask_cfpb.models.pages.SearchQuerySet.filter')
     def test_category_page_context_no_es(self, mock_es_query):
+        blank_facets = {'answers': {},
+                        'audiences': {},
+                        'subcategories': {}}
         mock_return_value = mock.Mock()
-        mock_return_value.facet_map = json.dumps(
-            {'answers': {},
-             'audiences': {},
-             'subcategories': {}})
+        mock_return_value.facet_map = json.dumps(blank_facets)
```

Comment:
Should `facet_map` here be `facets`?

Reply:
facet_map represents the json string that the elasticsearch query returns.
we deserialize that to make the `facets` value in context.